### DISCUSSION TITLE
Reset paginated table whilst staying on the same page?

### DISCUSSION THREAD

**Discussion by berwynn** [livewire/livewire#2698] [1. Help]
Hey all!

So I figured out a way to reset a paginated table by editing the WithPagination.php trait that comes with Livewire.
It looks like this : 
```
public function resetPage($pageNumber = 1)
    {
        $this->setPage($pageNumber);
    }
```

Then I use the method like so: 
```
public function resetTable() {
        $this->resetPage($this->page); //page no. pulled in from trait
    }
```

This is all well and good, but this isn't being tracked by github. Therefore my question is this: Is there a way to edit this trait and make it trackable via source control so it can be updated on all other branches too?

**Comment by joshhanley** [comment]
@WS243593 you could just make a copy of the trait and save it into your project somewhere and `use` your modification of it instead of the Livewire one.

Or if you only need it in one component, just `use` the trait as per normal and override the `resetPage` method inside your component.

Hope this helps!

**Reply by berwynn** [reply]
Hey Josh, 

Thank you very much for your reply. I didn't think of doing either of those things. I ended up removing the file from git tracking and it worked that way, though I need to remember during merges to exclude it otherwise it could reset itself. :D
